rules/Php80/Rector/ClassMethod/AddParamBasedOnParentClassMethodRector.php

```diff
@@ -239,6 +240,21 @@ function (Node $subNode) use ($parentClassMethodParam): bool {
         return $node;
     }
 
+    private function resolveParamDefault(Expr $expr): Expr
+    {
+        // re-create to avoid TokenStream error
+        if ($expr instanceof Array_) {
+            return new Array_($expr->items);
+        }
+
+        if ($expr instanceof String_) {
```

Comment:
I wonder, when we need a special case for string here, do we also need one for int/float?

or would the string case also be handled by the BetterStandardPrinter?

https://getrector.com/demo/6bf890b2-8642-421a-bda6-e9b6914af37b

Reply:
I updated to ensure printed first https://github.com/rectorphp/rector-src/pull/4833/commits/620db90fb0c57eae4190862553345829cd0ae3f1

Identifier can't be empty `''` or `[]`

Reply:
in case we don't have a test for the int/float case and it crashes as can be seen for the above link, we should add a test-case

Reply:
Oh, I see, I will add test for it.

Reply:
I added failing tset case for default int https://github.com/rectorphp/rector-src/pull/4833/commits/94389713244396c00c374ee8c12981fa8df0d9de

Reply:
Fixed :) https://github.com/rectorphp/rector-src/pull/4833/commits/525442f088545e243430a9e85fae1760d83b90a2

Reply:
Thanks